CHANGELOG.md

```diff
@@ -3,10 +3,20 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
-## Unreleased
+## 2.0.5 - 2017-12-14
 
 ### Added
 - Support for security list rule source port ranges (#340). This can be specified in "tcp_options" and "udp_options" using "source_port_range".
+- Allow configuration of default resources under VCNs (#374). See more details about this feature [here](https://github.com/oracle/terraform-provider-oci/blob/master/docs/Managing%20Default%20Resources.md)
+
+### Fixed
```

Comment:
Add fix for issue 170 https://github.com/oracle/terraform-provider-oci/commit/ea9e254bfcb50f38ca8faccef52e8daa583dffbb#diff-986abfb66fbe4ec940da975ae72b749b

"Fix error where same retry token was being used for multiple requests in some development environments when auto retries were activated (#170)"